index.html

```diff
@@ -37,7 +37,7 @@
     <div class="hero-unit" id="hero">
     <div style="width: 700px">
     <h1>Exposure checker</h1>
-    <p>Look at LSST images. Discover flaws we would otherwise have missed.<br>Make our data better!</p>
+    <p>Look at simulated LSST images. Discover flaws we would otherwise have missed.<br>Make our simulations and data processing better!</p>
```

Comment:
I fully imagine that we will also use the exposure checker for real images. I think it's fine to change it to "simulated" for now, but we should be prepared to change it back.